CK3ToEU4Tests/CK3WorldTests/ReligionTests/FaithTests.cpp

```diff
@@ -40,19 +41,21 @@ TEST(CK3World_FaithTests, faithPrimitivesCanBeLoaded)
 	input << "desc = \"Custom Desc\"\n";
 	input << "icon = \"gfx/icon.dds\"\n";
 	input << "template = \"catholic\"\n";
+	input << "variables = { \"a bunch of nonsense \nreally\"}";
 
 	const CK3::Faith faith(input, 42);
 
 	ASSERT_EQ("akom_pagan", faith.getName());
 	ASSERT_EQ(commonItems::Color(std::array<int, 3>{229, 178, 76}), faith.getColor());
 	ASSERT_EQ(3, faith.getDoctrines().size());
-	ASSERT_EQ(1, faith.getDoctrines().count("tenet_adorcism"));
-	ASSERT_EQ(1, faith.getDoctrines().count("doctrine_monogamy"));
-	ASSERT_EQ(1, faith.getDoctrines().count("doctrine_deviancy_shunned"));
+	ASSERT_EQ(1, faith.getDoctrines()[0] == "tenet_adorcism");
```

Comment:
ASSERT_TRUE(faith.getDoctrines().contains("tenet_adorcism"));
same for those below.